main.go

```diff
@@ -142,9 +145,15 @@ func main() {
 		glog.Info("Postgres metrics connection string was not set, postgres metrics are disabled.")
 	}
 
+	vodDecryptPrivateKey, err := crypto.LoadPrivateKey(cli.VodDecryptPrivateKey)
+
+	if err != nil {
+		glog.Fatalf("Error loading vod decrypt private key: %v", err)
+	}
+
```

Comment:
WDYT of adding a logic like this to validate that the public and private keys are compatible?
```suggestion
	publicKey, err := x509.ParsePKCS1PublicKey([]byte(cli.VodDecryptPublicKey))
	if err != nil {
		glog.Fatalf("Error parsing vod decrypt public key: %v", err)
	}
	if !publicKey.Equal(vodDecryptPrivateKey.Public()) {
		glog.Fatalf("Public key does not match private key")
	}
```

Or we could have a helper in `crypto` package like `crypto.ValidatePublicKey(...) error` or something.

Or... Maybe we don't need the public key CLI arg, can just generate it from the private key provided.